### ISSUE TITLE
No module named 'clickgen.libs'

### ISSUE THREAD

**Issue by yochananmarqos** [ful1e5/clickgen#60]
I'd be happy to update the AUR package to 2.1.5, however...

```
❯ ctgen --help
Traceback (most recent call last):
  File "/usr/bin/ctgen", line 5, in <module>
    from clickgen.scripts.ctgen import main
  File "/usr/lib/python3.11/site-packages/clickgen/scripts/ctgen.py", line 16, in <module>
    from clickgen.configparser import parse_config_file
  File "/usr/lib/python3.11/site-packages/clickgen/configparser.py", line 12, in <module>
    from clickgen.libs.colors import print_warning
ModuleNotFoundError: No module named 'clickgen.libs'
```

Am I missing something?

**Comment by ful1e5** [comment]
@yochananmarqos Fixed in [v2.1.6](https://github.com/ful1e5/clickgen/releases/tag/v2.1.6).

**Comment by yochananmarqos** [comment]
Thanks. 

However, I'm not able to build Bibata: https://github.com/ful1e5/Bibata_Cursor/issues/143
